server/src/controllers/eventController.ts

```diff
@@ -743,7 +743,7 @@ eventController.put('/', async (req: Request, res: Response) => {
         eventid: Number(req.body.eventid),
       },
       data: {
-        seasonid_fk: Number(req.body.seasonid_fk),
```

Comment:
@benfaustin95 Converting to a Number caused the update events endpoint to throw an error when a NULL value was used for season id. Not sure if there are any major side effects with this change, will like a second opinion.